vmp/resource_rulesengine_v4policy.go

```diff
@@ -166,41 +134,85 @@ func resourcePolicyRead(ctx context.Context, d *schema.ResourceData, m interface
 	}
 
 	json := string(jsonBytes)
-	InfoLogger.Printf("Retrieved policy: %s\n", json)
+
+	InfoLogger.Printf("Retrieved policy from API: %s\n", json)
 	d.Set("policy", json)
 
 	return diags
 }
 
 func resourcePolicyUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	diags := resourcePolicyCreate(ctx, d, m)
-
-	return diags
+	return resourcePolicyCreate(ctx, d, m)
 }
 
 func resourcePolicyDelete(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	var diags diag.Diagnostics
-	providerConfiguration, err := m.(*ProviderConfiguration).ApplyAccountNumberOverride(d.Get("account_number").(string))
+
+	policy := d.Get("policy").(string)
+
+	// pull out platform from existing policy
+	policyMap := make(map[string]interface{})
+	json.Unmarshal([]byte(policy), &policyMap)
+
+	platform := policyMap["platform"].(string)
+
+	// You can't actually delete policies, so we will instead
+	// create a placeholder empty policy for the customer for the given platform and policy type
+	emptyPolicy := map[string]interface{}{
```

Comment:
I can probably change this to a hard coded string and use string format to add the title